dnf-behave-tests/dnf/install-remove.feature

```diff
@@ -311,4 +311,27 @@ Scenario: User-installed packages are not removed as unused dependencies
         | remove-unused | sugar-0:1.0-1.x86_64              |
 
 
+@dnf5daemon
+Scenario: Install with glob
+   When I execute dnf with args "install 'water_*'"
+   Then the exit code is 0
+    And Transaction is following
+        | Action        | Package                           |
+        | install       | water_still-0:1.0-1.x86_64        |
+        | install       | water_carbonated-0:1.0-1.x86_64   |
+   When I execute dnf with args "remove 'water_*'"
+   Then the exit code is 0
+    And Transaction is following
+        | Action        | Package                           |
+        | remove        | water_still-0:1.0-1.x86_64        |
+        | remove        | water_carbonated-0:1.0-1.x86_64   |
+
+
+@dnf5daemon
+# https://github.com/rpm-software-management/dnf5/issues/766
+Scenario: Install with glob and empty advisory
+   When I execute dnf with args "install 'water_*' --advisories=nonexistent"
```

Comment:
This is interesting. Not really related to the test but seeing this I am wondering if we shouldn't report an error (or print some warning) if the user specifically asks for an advisory by its name and its not found.

Like if I wanted to make sure I am safe from some critical vulnerability, then making a typo in the advisory name could be pretty misleading.

Reply:
You're right, we should, and DNF 4 was much more clear about not finding any matches:

```
evan@fedora ~ $ sudo dnf install --advisory=foo hello
Last metadata expiration check: 0:00:52 ago on Thu 27 Jul 2023 01:16:07 PM UTC.
No security updates for argument "hello"
All matches were filtered out by exclude filtering for argument: hello
Error: Unable to find a match: hello
```

Reply:
I have made an issue: https://github.com/rpm-software-management/dnf5/issues/784
Thanks